how-to-pitch/readme.md

```diff
@@ -0,0 +1,64 @@
+
+### How to tell your story
+While telling your story there are a few key points you need to follow:
+
+* There needs to be a natural transition 
+    * Make sure the story you are telling progresses naturaly, if the listener needs to think back to what you told earlyer you will lose their attention.
+* It needs to be understandable
+    * Never use terms or principles without explaining them first, if there is a chance the listener won't  understand them.
+
+### Instilling trust in what you say and how you say it
+
+People buy who you are, not what you offer. Super cliche, but super true.  
+
+You can instill trust with your pitch if you try to use the follwing points while pitcing. Dont force it and dont take it to heavy.
+
+* Be integer:
+    * Are you really trying to achieve these goals or do you have alternative motives?" Easiest one: be honest. Don't try to be more than you are and most of all: don't be an ass.
+* Be passionate:
+    * Are you enthusiastic enough to go beyond the obvious?
+* Be Experienced:
+    * Are you capable of achieving your set goals? This can be as simple as dropping a line that suggests that you did this before.
+* Be Knowledgable:
+    * Do you understand what you are talking about? This can be done by clearly and simply explaining complicated factors of your pitch.
+* Be Skilled:
+    * Are you capable of achieving the promise, or can you hire the right people?
+* Show Leadership:
+    * Are you capable of creating and maintaining a team? This is mostly done with the way you talk and your posture. Can you demand and maintain their attention.
+* Show Commitment:
+    * Are you reliable to continue even if things are not going as pleased? This sounds like simply showing enthusiasm, but this is more about making clear that you have no trouble going for the long haul.
+* Show Vision:
+    * Are you capable to look beyond the obvious? If you solution/pitch is original enough you got this covered.
+* Be Realistic:
+    * Are you realistic or are you just spewing numbers to convince me?
+* Be Coachable:
+    * Will you take advice from your investor? Do you have the ability to listen?
+
+## Know the situation you are pitching in
+The most important thing is to understand the situation in which you are pitching. When you pitch for certain clients or events, you can do some research. But most of the time you will not know beforehand when you will need to tell your story. Understanding the situation you are in only comes with practice. Tell your story and see how people respond. Pitch to your parents, your friends, your partner, the guy in the grocery store that only buys one tube of toothpaste, your mirror, your pillow... Practice makes perfect and while you are practicing you will get the word out, which is always a good thing. The more you pitch the better you get to understand the situation you are in and how to use it in order to tell your pitch in the right way. 
```

Comment:
Think last sentence is too long and kind of double with the rest of paragraph. Maybe take it out and say something like "Good luck!" or whatever?